### ISSUE TITLE
Test mode

### ISSUE THREAD

**Issue by SinisterLuffy5** [MeiMeiSama/ACR#13]
Add a test mode, in which we can repick to test every hero.

**Comment by null15** [comment]
Good idea actually, will add it in the WIP, further on this at a later date
